### PR TITLE
Homework21: Pavel Mitrafanau

### PR DESCRIPTION
**Pull request by Pavel2812mi**
Good day, Sergey.
Made test cases (test folder) to check the operation of library and bank deposits modules (source folder, also i added all previous homework modules into it )
​​​Pylint error W0201 occurs in test_bank_deposit, I haven’t found a way to fix it yet, so I temporarily added it to the exceptions

### CODE REVIEW

**Comment by Pavel2812mi** [comment]
HTML report with test results added to the project tests folder

**Comment by Pavel2812mi** [comment]
Fixed pylint E0401 error, all is correct now

File: homework21/tests/test_library.py [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+def setup_logging(test_name, log_level="DEBUG"):
+    """
+    Logger setup.
+    """
+    logger.remove()
+    log_dir = os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                           "library_test_logs")
+    os.makedirs(log_dir, exist_ok=True)
+    log_file = os.path.join(log_dir, f"{test_name}.log")
+    logger.add(log_file, level=log_level, enqueue=True)
```
Comment (Stml89) [inline homework21/tests/test_library.py]: You can use 1 logger across a single application, e.g.: https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by Pavel2812mi** [comment]
Added single logger for test_library and for test_bank_deposit modules

File: homework21/tests/test_library.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+from loguru import logger
+from ..source.Homework11_1 import Reader, Book
+
+
+log_dir = os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                       "library_log_folder")
+os.makedirs(log_dir, exist_ok=True)
+log_file = os.path.join(log_dir, "library_test.log")
+logger.remove()
+logger.add(log_file, level="DEBUG", enqueue=True)
```
Comment (Stml89) [inline homework21/tests/test_library.py]: It is better this time, but I saw the same lines in `homework21/tests/test_bank_deposit.py` file. Better to use 1 file where you setup a logger, in test modules you will ned hust import it and use
https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by Pavel2812mi** [comment]
Made silngle logger file in tests folder with logging settings for both test modules

**Review by Stml89 — APPROVED** [review]

**Pull request merged**